tests/lib/hardware/hardware_base.py

```diff
@@ -114,7 +114,7 @@ def node_remove(self, node: NodeBase):
         node.destroy()
 
     @abstractmethod
-    def boot_nodes(self, masters: int = 1, workers: int = 2, offset: int = 0):
```

Comment:
Maybe we just skip the defaults here and make masters & workers mandatory?

Reply:
yes I thought about it. I'll change that.

Reply:
In this PR? Or another one?

Reply:
It is done in this one.
I removed defaults in function call but left default at the config level.